go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/mitchellh/go-testing-interface v1.14.0 // indirect
 	github.com/mitchellh/mapstructure v1.3.3 // indirect
 	github.com/prometheus/client_golang v1.9.0
+	github.com/prometheus/common v0.15.0
```

Comment:
This dependency should now be superflous -- one `go mod tidy` should fix this. ;)